azure/defaults.go

```diff
@@ -139,11 +139,6 @@ func GeneratePrivateDNSZoneName(clusterName string) string {
 	return fmt.Sprintf("%s.capz.io", clusterName)
 }
 
-// GeneratePrivateFQDN generates FQDN for a private API Server.
-func GeneratePrivateFQDN(clusterName string) string {
```

Comment:
instead of removing the func, could we change the parameters to take the dns zone name instead of the cluster name? Then it would be used in cluster.go like this:

```
azure.GeneratePrivateFQDN(s.GetPrivateDNSZoneName())
```

IMO it's slightly better than directly constructing the name with `fmt.Sprintf` in cluster.go

Reply:
Good point. Fixed, thanks!